src/components/evm/UserInfo.vue

```diff
@@ -0,0 +1,203 @@
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { useAccountStore, getAntelope } from 'src/antelope';
+import InlineSvg from 'vue-inline-svg';
+
+const accountStore = useAccountStore();
+const ant = getAntelope();
+
+export default defineComponent({
+    name: 'UserInfo',
+    components: {
+        InlineSvg,
+    },
+    methods: {
+        copyAddress() {
+            if (!accountStore.loggedAccount.isNative && accountStore.isAuthenticated) {
+                const address = accountStore.loggedEvmAccount.account;
+                navigator.clipboard.writeText(address.toUpperCase());
+                ant.config.notifySuccessHandler(this.$t('settings.copied_ok'));
+            }
+        },
+        logout() {
+            accountStore.logout();
+        },
+        toggleDarkMode() {
+            this.$q.dark.toggle();
+        },
+        gotoTeloscan() {
+            const explorerUrl = ant.stores.chain.loggedEvmChain?.settings.getExplorerUrl();
+            if (explorerUrl) {
+                window.open(explorerUrl + '/address/' + accountStore.loggedEvmAccount.account, '_blank');
+                return;
+            } else {
+                ant.config.notifyErrorHandler(this.$t('settings.no_explorer', { network: accountStore.loggedAccount.network }));
+            }
+        },
+    },
+    computed: {
+        address() {
+            if (accountStore.isAuthenticated) {
+                return accountStore.loggedEvmAccount.displayAddress;
```

Comment:
this should also be checksum not caps

Reply:
done ([6ed1f67](https://github.com/telosnetwork/telos-wallet/pull/220/commits/6ed1f6770f0b97f1fee1dc905517b08de7b3c59e))